extmod/lwip-include/lwipopts_common.h

```diff
@@ -65,7 +65,8 @@
 
 #define LWIP_NUM_NETIF_CLIENT_DATA      LWIP_MDNS_RESPONDER
 #define MEMP_NUM_UDP_PCB                (4 + LWIP_MDNS_RESPONDER)
-#define MEMP_NUM_SYS_TIMEOUT            (LWIP_NUM_SYS_TIMEOUT_INTERNAL + LWIP_MDNS_RESPONDER)
+#define _MDNS_TIMERS                    (2 + (5 * (LWIP_IPV4 + LWIP_IPV6)))
```

Comment:
I know this is a bit picky, but I think this macro should either be named longer like `MEMP_NUM_SYS_TIMEOUT_MDNS`, or not defined at all and just write the expression inline in `MEMP_NUM_SYS_TIMEOUT`.

Reply:
What do you think about the leading underscore? I wanted to make it clear it was not an lwIP flag.